ecs-init/config/common.go

```diff
@@ -99,3 +117,28 @@ func DockerUnixSocket() (string, bool) {
 	// eg: Docker daemon restart
 	return "/var/run", false
 }
+
+// ec2MetadataRegion finds the Region name from Metadata. If an error occurs fetching the region the default region name
+// is returned.
+func ec2MetadataRegion () string {
```

Comment:
Name this `getRegion`.  It's an implementation detail that it comes from EC2 instance metadata.